### PR TITLE
Adding maharshi-sinha.json

### PR DESCRIPTION
**Pull request by maharshi-sinha**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3493"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
hey you should remove all the files except  `maharshi-sinha.json`  from the commit

File: public/data/maharshi-sinha.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      "name": "Read my blogs on Hashnode",
+      "url": "https://maharshisinha.hashnode.dev",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Check out my website",
+      "url": "https://maharshi-sinha.github.io/portfolio-site/",
+      "icon": "globe"
+    }
+  ]
```
Comment (Aadarsh805) [inline public/data/maharshi-sinha.json:33]: and there is a missing comma at the end fo line 33

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
All the icons are wrong, search icons `https://linkfree.eddiehub.io/icons` here

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
You have to remove all the files except maharashi-sinha.json

**Review by Aadarsh805 — COMMENTED** [review]
I see that you have 5 files in the commit, you should remove them except `maharshi-sinha.json`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Also your file is in wrong directory it should be `data/maharshi-sinha.json`

**Comment by Aadarsh805** [comment]
If you need help with the deleting task, feel free to ask okay

**Pull request closed without merging**